docs/sources/search/_index.md

```diff
@@ -83,3 +83,22 @@ To filter dashboard search result by a tag, complete one of the following steps:
   All tags will be shown, and when you select a tag, the dashboard search will be instantly filtered.
 
 > **Note:** When using only a keyboard, press the `tab` key and navigate to the **Filter by tag** drop-down menu, press the down arrow key `▼` to activate the menu and locate a tag, and press `Enter` to select the tag.
+
+# Command Palette
+
```

Comment:
I recommend adding an introduction here that describes what the command palette is and why the user should care. What are the benefits?

Reply:
@chri2547 After making the change you mention in your following suggestion, should I still add an introduction or will it be repetitive? Thanks in advance!